app/models/invite.js

```diff
@@ -13,33 +12,18 @@ export default Model.extend({
     updatedAtUTC: attr('moment-utc'),
     updatedBy: attr('number'),
     status: attr('string'),
-    roles: hasMany('role', {
-        embedded: 'always',
-        async: false
-    }),
+    role: belongsTo('role', {async: false}),
 
     ajax: injectService(),
     ghostPaths: injectService(),
 
-    role: computed('roles', {
-        get() {
-            return this.get('roles.firstObject');
-        },
-        set(key, value) {
-            // Only one role per user, so remove any old data.
-            this.get('roles').clear();
-            this.get('roles').pushObject(value);
-
-            return value;
-        }
-    }),
-
     resend() {
         let fullInviteData = this.toJSON();
         let inviteData = {
             email: fullInviteData.email,
-            roles: fullInviteData.roles
+            role: fullInviteData.role
```

Comment:
No, here we're using ajax directly:

```
let inviteUrl = this.get('ghostPaths.url').api('invites');

        return this.get('ajax').post(inviteUrl, {
            data: JSON.stringify({invites: [inviteData]}),
            contentType: 'application/json'
        });
```

It's not going through Ember Data and so none of the models or serialisers are touched.